### PR TITLE
feat/live-agent-integration-wiki

### PR DESCRIPTION
**Pull request by itIsJim**
Missing the following sections
- Channel: 
  - [ ] Messenger
  - [ ] WhatsApp
  - [x] Line
- Integration:
  - [ ] Google Calendar
  - [x] Shopify
  - [x] Squarespace

### CODE REVIEW

File: content/en/seachat/seachat-manual/06-live-agent-transfer.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+date: 2020-10-06T08:48:45+00:00
+lastmod: 2020-10-06T08:48:45+00:00
+weight: 70
+draft: false
+images: []
+toc: true
+---
+A live agent is a human agent who can take over the conversation from the chatbot agent when needed. Not only can live agents take over the conversation, but they can also provide the virtual agent with new knowledge, and help train the virtual agent by testing and fine-tuning the response of the chatbot agent.
+
+Depending on your SeaChat integration, it might take different configurations to enable the live agent feature. SeaChat offers many integration configurations and channels. We recommend you to refer to [Integration](/en/seachat/seachat-manual/integrations/) or [Channels](/en/seachat/seachat-manual/channels/) to learn how to set up your agent properly.
```
Comment (amy17519) [inline content/en/seachat/seachat-manual/06-live-agent-transfer.md:14]: Depending on your SeaChat **channel**

File: content/en/seachat/seachat-manual/06-live-agent-transfer.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+<br/>
+<a href="/images/product-updates/seachat/en/live-agent-transfer/20240325-live-agent-transfer-3.png" target="_blank">
+<img width="100%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/product-updates/seachat/en/live-agent-transfer/20240325-live-agent-transfer-3.png" alt="">
+</a>
+<br/>
+
+Once the live agent has completed the conversation, the conversation will be marked as completed and the customer will be notified that the conversation has been completed. Live agent can also reactivate the conversation when needed. Simply click on the **Reactivate** button to restart the conversation.
+
+
```
Comment (amy17519) [inline content/en/seachat/seachat-manual/06-live-agent-transfer.md:56]: Add sections: 
- How to Request Live Agent on Webchat
Add notes that the Live Agent Transfer works with Shopify and Squarespace. They are essentially webchat
- How to Request Live Agent on LINE
Jun Mao Image Mar 26

- How to Request Live Agent on WhatsApp
@xuchen screenshot?
- How to Request Live Agent on Messenger
@xuchen screenshot?

Comment (itIsJim) [reply content/en/seachat/seachat-manual/06-live-agent-transfer.md:56]: we will do WhatsApp and Messenger on separate PRs

File: content/en/seachat/seachat-manual/06-live-agent-transfer.md [hunk 3]
```diff
@@ -64,4 +71,16 @@ The chat conversation will let your customer know how many live agents are curre
 <a href="/images/product-updates/seachat/en/live-agent-transfer/20240325-live-agent-transfer-webpage-1.png" target="_blank">
 <img width="100%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/product-updates/seachat/en/live-agent-transfer/20240325-live-agent-transfer-web-widget-1.png" alt="">
 </a>
-<br/>
\ No newline at end of file
+<br/>
+
+
+## LINE Channel
```
Comment (amy17519) [inline content/en/seachat/seachat-manual/06-live-agent-transfer.md:79]: ## How to Request Live Agent on LINE

**Review by amy17519 — APPROVED** [review]

**Pull request merged**